### DISCUSSION TITLE
More error information

### DISCUSSION THREAD

**Discussion by vjpr** [oven-sh/bun#1478] [Q&A]
I'm getting some Bun errors such as the following with no stack trace as to where the error comes from.

```
TypeError: Cannot call a class constructor without |new|
```

Tried `logLevel = "debug"` in a `bunfig.toml` and it didn't work.

**Comment by Jarred-Sumner** [comment]
Is this in canary? There was a bug that broke stacktraces but has since been fixed

**Comment by rushi** [comment]
I got a similar error with **bun v1.0**, and this stack trace. I had no idea what module was causing this till I searched for `class AsyncResource` and found it was from the Inquirer module which is doing `import { AsyncLocalStorage, AsyncResource } from 'node:async_hooks';` 

```js
121 | class AsyncResource {
122 |   type;
123 |   #snapshot;
124 |
125 |   constructor(type, options) {
                             ^
TypeError: Cannot call a class constructor without |new|
      at AsyncResource (node:async_hooks:125:26)
      at node:readline:114:54
      at onData (node:readline:476:34)
      at addChunk (node:stream:1940:42)
      at readableAddChunk (node:stream:1894:58)
      at processTicksAndRejections (:1:2602)
```
